packages/deprecation-crawler/src/lib/utils.ts

```diff
@@ -153,6 +157,15 @@ export function semverSort(
     return semvers.sort(function (v1, v2) {
       const p1 = pick(v1);
       const p2 = pick(v2);
+      if (p1 === p2) {
+        return 0;
+      }
+      if (!p1) {
```

Comment:
When we crawl without adding a version to the deprecation the sorting would throw an error because the tag isn't following the semver convention. That's why I added these extra checks.